source/agora/test/Simple.d

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+
+    Contains the simplest possible block creating test
+
```

Comment:
Add the commit description (especially how to run it) here or in README of the test directory.